### PR TITLE
Rollback IE11 hacks

### LINKED ISSUE #494: Rollback IE11 hacks
After #493 we can rollback the several hacks we did to try to support IE11.

- [x] `initial` values were removed as part of https://github.com/CartoDB/airship/pull/491
- [x] parsing of `computedStyles` to extract colors from CSS variables. https://github.com/CartoDB/airship/pull/486/files
- [x] `padStart` polyfills https://github.com/CartoDB/airship/pull/487 (research if some other browsers need it)
- [x] IE11 histogram handles https://github.com/CartoDB/airship/pull/484
- [x] IE11 Remove javascript styles from widgets

Feel free to add whatever hack you are aware of.

Note: this is not urgent.

### LINKED ISSUE #516: [Edge] Y-Axis first value is not aligned with rest
On edge, the first value of the Y-Axis is not aligned with the rest. This behaviour does not happen with the histogram.

image

### PR DESCRIPTION
**Pull request by rjimenezda**
Fix #494 
Fix #516 
---

We should test this one on Edge, juuuust in case.

### CODE REVIEW

**Comment by rjimenezda** [comment]
Yeah, let's wait until we test it on Edge. Just in case something is horribly broken.

**Comment by rjimenezda** [comment]
I've reverted part of the readable-number 'hack'. Because of #516.

Sorry about the force-pushes, had to rebase a couple of times 🙇

File: packages/components/src/components/as-histogram-widget/as-histogram-widget.tsx [hunk 1]
```diff
@@ -16,8 +16,10 @@ import { Container } from './types/Container';
 import dataService from './utils/data.service';
 import drawService from './utils/draw.service';
 
-const DEFAULT_BAR_COLOR = 'var(--as--color--primary, #1785FB)';
-const DEFAULT_SELECTED_BAR_COLOR = 'var(--as--color--complementary, #47DB99)';
+const DEFAULT_BAR_COLOR_HEX = '#1785FB';
```
Comment (ivanmalagon) [inline packages/components/src/components/as-histogram-widget/as-histogram-widget.tsx]: Is it impossible to get this color from the `scss` file? It'd be great to have all colors sync'ed.

At least add a comment to tell that it's the primary color.

Comment (ivanmalagon) [reply packages/components/src/components/as-histogram-widget/as-histogram-widget.tsx]: 👍

**Pull request merged**